lib/cocoapods/installer.rb

```diff
@@ -145,6 +164,31 @@ def install!
       perform_post_install_actions
     end
 
+    def analyze_project_cache
+      object_version = aggregate_targets.map(&:user_project).compact.map { |p| p.object_version.to_i }.min
```

Comment:
can you do:
```ruby
aggregate_targets.map(&:user_project).compact.min(&:object_version)
```
instead?

Reply:
maybe that is a 2.6 version only but you can do `min`

Reply:
`min_by`